pkg/eval/unix/unix.go

```diff
@@ -0,0 +1,11 @@
+// Package `unix` exposes variables and functions that deal with features
+// unique to UNIX like operating systems. On other operating systems it will
+// be an empty namespace; or have a subset of the features normally available
+// on a UNIX like operating system.
+package unix
+
+import (
+	"github.com/elves/elvish/pkg/eval"
+)
+
+var Ns = eval.Ns{}
```

Comment:
I think it's cleaner to declare the namespace literally here, and have another `unix_non_unix.go` that declares an empty namespace.

Reply:
Okay, that works for me. But it does mean the `// +build` directive will need to be updated when new platforms are supported by Go. And we'll want to proactively include the `js` platform name.